cves/CVE-2013-2277.yml

```diff
@@ -64,7 +64,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulenrability was in the ff_h264_decode_seq_parameter_set function in
+  h264_ps.c in the way that it does not validate the relationship between the
+  depths of luma and chroma. In video, luma represents the brightness of the
+  pixel and chroma represents the color information. They are typically paired
+  up. The existing check in libavcodec in FFmpeg before 1.1.3 is not validating
+  if the depths of luma and chroma do not match. 
+
+  The way this could be exploited is that an attacker can cause a denial of
+  service due to out-of-bounds array access. At best, the application will
+  crash. At worst, there will be unspecified impacts through crafted H.264 data.
```

Comment:
What is H.264 data?

Reply:
H.264 is widely used codec with penetration in streaming video markets, broadcast, etc. Codec is a device or program that compresses data to enable faster transmission and decompresses received data. I would add this to the document and update it.